website/content/en/status/report-2023-04-2023-06/openstack.adoc

```diff
@@ -1,30 +1,30 @@
 
-Links: + 
-link:https://www.openstack.org/[OpenStack] URL: link:https://www.openstack.org/[] + 
-link:https://github.com/openstack-on-freebsd[OpenStack on FreeBSD] URL: link:https://github.com/openstack-on-freebsd[] + 
+Links: +
+link:https://www.openstack.org/[OpenStack] URL: link:https://www.openstack.org/[] +
+link:https://github.com/openstack-on-freebsd[OpenStack on FreeBSD] URL: link:https://github.com/openstack-on-freebsd[]
 
-Contact: Chih-Hsin Chang <starbops@hey.com> + 
-Contact: Li-Wen Hsu <lwhsu@FreeBSD.org> 
+Contact: Chih-Hsin Chang <starbops@hey.com> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org>
 
-This project aims to port key OpenStack components such as keystone, nova, neutron, etc., so that FreeBSD can function as an OpenStack host. 
+This project aims to port key OpenStack components such as keystone, nova, and neutron, so that FreeBSD can function as an OpenStack host.
 
-In 2023 Q2, we start porting `nova-novncproxy` and `nova-serialproxy` to increase the ways to access the instance console. 
-To lower the threshold for people who want to give it a try on the project, we also migrate our development environment from a physical machine to a virtual one. 
-But there is still a problem running bhyve VMs on top of Linux KVM. 
-A detailed writeup for the issue can be found link:https://hackmd.io/@starbops/SkdJON2un[here]. 
-Other achievements include: 
+We started porting `nova-novncproxy` and `nova-serialproxy` to increase the ways to access the instance console.
+To lower the threshold for people who want to give it a try on the project, we also migrated our development environment from a physical machine to a virtual one.
+There is still a problem running bhyve VMs on top of Linux KVM.
+A detailed writeup for the issue can be found link:https://hackmd.io/@starbops/SkdJON2un[here].
+Other achievements include:
 
-* Sorting out network connectivity issues inside the instances 
-* Able to spawn multiple instances 
-* Porting from Python 3.8 to 3.9 
+* Sorting out network connectivity issues inside the instances
+* Able to spawn multiple instances
+* Porting from Python 3.8 to 3.9.
 
-In the next quarter, we will continue working on the console proxy services to make the overall workflow more fluent. 
+In the next quarter, we will continue working on the console proxy services to make the overall workflow more fluent.
 
-The step-by-step documents for constructing a POC site can also be found link:https://github.com/openstack-on-freebsd/docs[in the `docs` repository]. 
-The patched version of each OpenStack component is under the same GitHub organization. 
+The step-by-step documents for constructing a POC site can also be found link:https://github.com/openstack-on-freebsd/docs[in the `docs` repository].
+The patched version of each OpenStack component is under the same GitHub organization.
 
 People interested in helping with the project can first help check the documentation by following the installation guide. 
```

Comment:
I think there is an extra space at the end of this line too.

Reply:
Resolved by [`5a643e2` (#209)](https://github.com/freebsd/freebsd-doc/pull/209/commits/5a643e25274e89c87c2eed9f7a512b8154ca2bc6)